### PR TITLE
Support instructable_embed endpoint

### PR DESCRIPTION
**Pull request by WoytenAA**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7.6.0
+
+- Add `instructable_embed` to `Client` and `AsyncClient`
```
Comment (fgebhart) [inline Changelog.md:5]: Would be great if you could add a bit more context, e.g. what this endpoint is used for. In the end this changelog is user facing.

File: aleph_alpha_client/embedding.py [hunk 2]
```diff
@@ -272,6 +269,66 @@ def _asdict(self) -> Mapping[str, Any]:
...
+    Parameters:
+        input
+            The text and/or image(s) to be embedded.
+        instruction
+            An instruction specifying the aspect to attend to when generating the embedding.
+        normalize
+            Return normalized embeddings. This can be used to save on additional compute when applying a cosine similarity metric.
+
+            Note that at the moment this parameter does not yet have any effect. This will change as soon as the
+            corresponding feature is available in the backend
```
Comment (fgebhart) [inline aleph_alpha_client/embedding.py:286]: While I see that this was just C&P, does this still hold true? Is it not yet supported by the backend?

File: tests/test_embed.py [hunk 3]
```diff
@@ -105,6 +106,24 @@ async def test_batch_embed_semantic_invalid_batch_sizes(
         await async_client.batch_semantic_embed(request=request, batch_size=batch_size)
 
 
+@pytest.mark.system_test
```
Comment (fgebhart) [inline tests/test_embed.py:109]: Did you add this decorator intentionally? I'm not sure whether we have this model in our system test setup. Could you double check?

File: tests/test_embed.py [hunk 4]
```diff
@@ -214,6 +233,23 @@ def test_embed_semantic(sync_client: Client):
     assert result.num_tokens_prompt_total >= 1
 
 
+@pytest.mark.system_test
```
Comment (fgebhart) [inline tests/test_embed.py:236]: same here

File: aleph_alpha_client/embedding.py (hunk 2 above)
Comment (WoytenAA) [reply aleph_alpha_client/embedding.py:286]: I will check. Thanks for pointing that out!

File: tests/test_embed.py (hunk 3 above)
Comment (WoytenAA) [reply tests/test_embed.py:109]: As far as I can tell, the test runs against the public AA API which has the required model deployed.

File: Changelog.md (hunk 1 above)
Comment (WoytenAA) [reply Changelog.md:5]: I just followed the existing changelog format which is rather brief. Details about the new function including an example can be found in the doc strings.

**Review by fgebhart — APPROVED** [review]

**Pull request merged**